src/main/resources/application.properties

```diff
@@ -4,7 +4,7 @@ spring.jpa.generate-ddl=true
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL95Dialect
 spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
-spring.datasource.username=postgres
+spring.datasource.username=yuliya
```

Comment:
let's use one username for everybody in the team(it can be yuliya, but you need to convince your teammates =) )